### PR TITLE
✨Populate the repo

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Part of https://github.com/kcp-dev/kcp/issues/1943
Part of https://github.com/kcp-dev/kcp/issues/209

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Copyright 2021 The KCP Authors.
```
Comment (ncdc) [inline Makefile:1]: 2022

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	k8s.io/klog/v2 v2.70.1 // indirect
+	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed // indirect
+	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
+	sigs.k8s.io/yaml v1.3.0 // indirect
+)
+
+replace github.com/kcp-dev/apimachinery => /home/stevekuznetsov/code/kcp-dev/apimachinery/src/github.com/kcp-dev/apimachinery
```
Comment (ncdc) [inline go.mod:41]: fix 😄

Comment (stevekuznetsov) [reply go.mod]: oopsie doopsie

**Pull request merged**